Makefile

```diff
@@ -172,6 +172,10 @@ ifeq ($(WITH_LXD), true)
   BUILD_TAGS+=lxd
 endif
 
+ifeq ($(WITH_LIBVIRT), true)
+  BUILD_TAGS+=libvirt
+endif
```

Comment:
Will not be activated by default. On the other hand, it should only be activated on Linux.

Reply:
it's hard to say, in my environment we have two separated jenkins jobs: build analyzer and agent separately, it's not mandatory for analyzers to have a libvirt installed on the system, it depends on your architecture, etc, so it's better to have a possibility to compile with libvirt support on demand.